samples/matter/common/src/bridge/matter_bridged_device.cpp

```diff
@@ -62,17 +63,42 @@ CHIP_ERROR MatterBridgedDevice::HandleReadBridgedDeviceBasicInformation(Attribut
 	}
 }
 
-CHIP_ERROR MatterBridgedDevice::HandleReadDescriptor(AttributeId attributeId, uint8_t *buffer, uint16_t maxReadLength)
+CHIP_ERROR MatterBridgedDevice::HandleWriteIdentify(chip::AttributeId attributeId, void *data, size_t dataSize)
 {
 	switch (attributeId) {
-	case Clusters::Descriptor::Attributes::ClusterRevision::Id: {
-		uint16_t clusterRevision = GetDescriptorClusterRevision();
+	case Clusters::Identify::Attributes::IdentifyTime::Id:
+		if (data && dataSize == sizeof(mIdentifyTime)) {
+			mIdentifyTime = (*reinterpret_cast<decltype(mIdentifyTime) *>(data));
```

Comment:
There's no guarantee `data` is aligned.  This should use `memcpy`, instead.

Reply:
Yes, thanks for the catch. I will fix it in the followup PR.